doc: High-Performance Framework (#21539)
nrfconnect/sdk-nrf: applications/hpf/gpio/README.rst

.. _sdp_gpio:
.. _hpf_gpio_example:

High-Performance Framework GPIO
###############################

.. caution::

   The High-Performance Framework (HPF) support in the |NCS| is :ref:`experimental <software_maturity>` and is limited to the nRF54L15 device.

> This application demonstrates how to write a High-Performance Framework (HPF) application and communicate with it.

Reviewer: How about
```suggestion
This application demonstrates how to write a High-Performance Framework (HPF) application implementing a simple peripheral.
```

> This application demonstrates how to write a High-Performance Framework (HPF) application and communicate with it.

Reviewer 2: applied
